README.rst

```diff
@@ -90,7 +90,7 @@ The following Python modules are required:
 
 Optional requirements include:
 
-* azure_ for Microsoft Azure object storage
+* azure_ for Microsoft Azure object storage (>= 5.0.0)
```

Comment:
This could mention that a patched version is required with a link to the patches